### PR TITLE
Update GSRenderer.cpp

### PR DESCRIPTION
**Pull request by jinwk00**
Brought from https://github.com/john-peterson/pcsx2/commit/d904952ec189eafc4b615e9a3e03b9a9d0d3f396
I TAKE NO CREDITS

### CODE REVIEW

**Comment by ssakash** [comment]
this looks like a hack, would probably cause issues on some other games which share the following scenarios. also you need a rebase. the forked branch you're trying to merged is actually 3 YEARS OLD.

also did you check whether Time Crisis 2, 3 split screen view works fine on the latest development builds ? http://pcsx2.net/download/development/git.html

**Comment by jinwk00** [comment]
Nope. It did not worked at all.

**Comment by ssakash** [comment]
Either way until the patch is being checked can you do a rebase and edit the commit message to something more meaningful on what the commit actually does ? I'll try getting the game and check out the source of the issue.

**Comment by gregory38** [comment]
Honestly without a real explanation it is a no-go. It smells like a hack for a specific game.

**Comment by danilaml** [comment]
Where are extra `+`

**Comment by jinwk00** [comment]
https://github.com/jinwk00/pcsx2/commit/0ca620a88ed5ad4d39f1e104e617bd68d34bfcd4
I actually made my own fork, so can anyone build it for me?

**Comment by ssakash** [comment]
> I actually made my own fork, so can anyone build it for me?

I'm not sure I understand you, do you mean you haven't tested whether the following commit changes fix splitscreen issues on this game ? did you only PR the changes after taking a look at the commit messages of the linked branch ? that's no good -_-

**Comment by jinwk00** [comment]
The thing is that I don't know how to test it out, because I am not like a genius in computer programming. Is it possible to test by another person?

**Comment by willkuer** [comment]
The proper way would probably be to open an issue recommending this solution. You need some compiler to compile your source code and test it in advance before you commit. If you don't know how to compile you can hardly help in the codebase i would say.

Maybe you try first to compile your source code. This probably needs a rebase beforehand (git rebase...). After you compiled you can test of the problem is solved. Then you should test a number of other games to check if there are regressions.

**Comment by refractionpcsx2** [comment]
if @john-peterson could give an explination of what he's doing and how it is accurate emulation and not a hack, it would be more considered, but right now it does just look like a hack.

**Comment by jinwk00** [comment]
Got it. I will try recompiling, and check if there is any problem. I
definitely need to show it up on the issues section.
2015. 12. 27. 오후 5:18에 "refractionpcsx2" <notifications@github.com>님이 작성:

File: plugins/GSdx/GSRenderer.cpp [hunk 1]
```diff
@@ -242,6 +242,12 @@ bool GSRenderer::Merge(int field)
 
 		fs.x = max(fs.x, (int)(dst[i].z + 0.5f));
 		fs.y = max(fs.y, (int)(dst[i].w + 0.5f));
+		// Fixing Time Crisis 2, 3 split screen view. Without this only the 1p screen is visible.
++		if(src[i].right - src[i].left == 0.5) {
++			src[i].left = 0;
++			src[i].right = 1;
++			dst[i].top += dst[i].bottom*0.25;
++			dst[i].bottom -= dst[i].bottom*0.25;
 	}
```
Comment (ssakash) [inline plugins/GSdx/GSRenderer.cpp]: you miss a ``}`` over here. also remove all the ``+`` from the code. to make it atleast compile on Appveyor so users could test whether it works :p

**Comment by gregory38** [comment]
I label is postponed because PR need more work. However without proper explanation it is a nogo.

**Comment by refractionpcsx2** [comment]
Gave it a couple more relevant tags.

**Comment by gregory38** [comment]
There are work (hopefully less hackish) on going on another PR so I'm closing this one.

**Pull request closed without merging**